plugins/inventory/kubevirt.py

```diff
@@ -448,6 +448,10 @@ def get_vmis_for_namespace(
                 f"Error fetching VirtualMachineInstance list: {self.format_dynamic_api_exc(exc)}"
             ) from exc
 
+        if len(vmi_list.items) < 1:
```

Comment:
```suggestion
        if not vmi_list.items:
```

Reply:
https://peps.python.org/pep-0008/#programming-recommendations

For sequences, (strings, lists, tuples), use the fact that empty sequences are false:

```
# Correct:
if not seq:
if seq:
# Wrong:
if len(seq):
if not len(seq):
```